### ISSUE TITLE
Cannot reproduce the results in the paper

### ISSUE THREAD

**Issue by CNJianLiu** [leo94-hk/SGPA#5]
Hi, thank you for open source this work, I retrained your model without changing the code, but the experimental results are quite different (some metrics differ by nearly 10 percent), I would like to know what caused this, the following are my environment settings:
Pytorch 1.7.0
Cuda 11.0.3
A single RTX3090

**Comment by PatrickFeng** [comment]
Hi Liu. Have you solved the performance gap issue?

**Comment by CNJianLiu** [comment]
> Hi Liu. Have you solved the performance gap issue?

Hi, I am sorry for the late reply. I got a drop in accuracy because of using 'camera_composed_depth' as training data.

**Comment by Ironman6666** [comment]
> > Hi Liu. Have you solved the performance gap issue?
> 
> Hi, I am sorry for the late reply. I got a drop in accuracy because of using 'camera_composed_depth' as training data.
Hi Liu.I also got the performence gap without changing the code.Did you finally get the accuracy shown in paper ? I didn't find anywhere about 'camera_composed_depth' in the code,so I don't know how to solve this problem.Can you give me some advice?
![image](https://user-images.githubusercontent.com/103632128/222885697-3441b6d1-c71d-4cc1-9339-147f598d16d4.png)

**Comment by CNJianLiu** [comment]
> 
Hi, I have got a similar accuracy shown in the paper. I used to use the depth data with background in the NOCS-CAMERA dataset for training, which led to a decrease in accuracy.

**Comment by Ironman6666** [comment]
> 
Tks for your nice reply! I'm sorry to bother again. I change the code in pose_dataset.py like below to use composed_full_depth in training according to your advice, but the accuracy didn't go up. Do I need more code modification? I'll very appreciate recieving your advice!
![image](https://user-images.githubusercontent.com/103632128/223033623-0add5541-e857-4efa-8488-52cca003ca96.png)
![image](https://user-images.githubusercontent.com/103632128/223033739-79206afe-eb00-4777-9b06-1a55bf998cfa.png)

**Comment by CNJianLiu** [comment]
> > 
> 
> Tks for your nice reply! I'm sorry to bother again. I change the code in pose_dataset.py like below to use composed_full_depth in training according to your advice, but the accuracy didn't go up. Do I need more code modification? I'll very appreciate recieving your advice! ![image](https://user-images.githubusercontent.com/103632128/223033623-0add5541-e857-4efa-8488-52cca003ca96.png) ![image](https://user-images.githubusercontent.com/103632128/223033739-79206afe-eb00-4777-9b06-1a55bf998cfa.png)

Hi, you may have misunderstood what I mean, the author used the depth map without background depth before. I lost accuracy due to using ‘composed_full_depth’.

**Comment by CNJianLiu** [comment]
> > 
> 
> Tks for your nice reply! I'm sorry to bother again. I change the code in pose_dataset.py like below to use composed_full_depth in training according to your advice, but the accuracy didn't go up. Do I need more code modification? I'll very appreciate recieving your advice! ![image](https://user-images.githubusercontent.com/103632128/223033623-0add5541-e857-4efa-8488-52cca003ca96.png) ![image](https://user-images.githubusercontent.com/103632128/223033739-79206afe-eb00-4777-9b06-1a55bf998cfa.png)

Hi, you should be able to solve this problem by running 'bash eval.sh', because the author did not use mask-rcnn for segmentation in the test program during training.
